hack/remove-sbr-finalizers.sh

```diff
@@ -7,11 +7,12 @@ else
 fi
 
 # Remove SBR finalizers if CRD exists
-[ -z "$(kubectl get -f deploy/crds/*crd.yaml -o jsonpath="{.metadata.name}" --ignore-not-found)" ] && exit 0
+CRD=$(kubectl get -f deploy/crds/*crd.yaml -o jsonpath="{.metadata.name}" --ignore-not-found)
+[ -z $CRD ] && exit 0
```

Comment:
`CRD` contains CRD name, not CRD itself. How about that we rename it ot `CRD_NAME`?

Reply:
Good point